chrome/content/zotero/containers/itemTree.jsx

```diff
@@ -0,0 +1,3430 @@
+/*
```

Comment:
Any particular reason these are in `containers`? Because they call privileged code?

I think we're mostly done with the container/component distinction. Most of the "components" aren't really going to be usable outside of Zotero past initial development, and having the collection tree and item tree not be "components" is mostly just going to be frustrating.

Reply:
There is value in a separation between React components that are divorced from any business logic and just display stuff. Containers/components is the current naming/paradigm. The files in the containers folder could just sit in the main zotero folder, although there might also be value in keeping the Reactified UI stuff separated for organization purposes?

Reply:
Display-only components make sense if we have something like https://github.com/zotero/zotero/issues/2030.

I don't see a lot of value in separating `containers`. Most of the things in the parent directory are already creating UI. React happens to be one technology we use, and we'll likely use Custom Elements for some things as well, but who cares? I think it's more important to not have to think about or know what specific technologies something is currently using and just be able to look at _a list of files that deal with UI with some business logic in them_.